src/option/option.ts

```diff
@@ -26,6 +26,30 @@ abstract class Option<T> implements Monad<T>, Matchable<T, undefined> {
     return new Some(value);
   }
 
+  /**
+   * Creates an `Option` instance from a value.
+   * @template T The type of the value.
+   * @param {NotNullable<T>} value The nullable value.
+   * @returns {Some<T>} A `Some` instance of the value
+   * @example
+   * const some = Option.ofSome(5);
+   * some.match(console.log, () => console.log('none')); // 5
+   */
+  static ofSome<T>(value: NotNullable<T>): Option<T> {
```

Comment:
Since we already have some factory method for creating specific instance of an abstract class, e.g. `Either.right(...)` and `Either.left(...)`, I would see more right if we call it `Option.some(...)` and `Option.none()`, just to be consistent in what we have.

Reply:
Changes applied!